### DISCUSSION TITLE
Just a "small" alert

### DISCUSSION THREAD

**Discussion by ghost** [Repl-it-Coders/Repl.it-2020#21] [General]
Based on recent activity. It seems that this organisation is being targeted with **certain** bots/users. I don't know why, but it does seem a little weird. I'll try looking into it further. @whippingdot has expressed suspicion in this a couple days before hand.

If you see any users mass pinging, for starters. Please ignore them. Also... don't click on shortened links anyways because you never know what is on the receiving end (like drive-by-download malware.)

Have a good year!

**Comment by ghost** [comment]
Targeted people include not only myself but the following:
 - @CoolCoderSJ 
 - @connordennison 
 - @cursorweb 
 - @Subroutine7901 
 - @tsunami-or-something 
 - @whippingdot 

There may be more matches.

**Reply by whippingdot** [reply]
Thanks for creating this discussion as a heads-up. It might be useful for those who might be vulnerable to pressing on those links or those who are confused lol.

**Comment by whippingdot** [comment]
If you do suppose get mentioned in a discussion / pr / issue with several others with a link, please do report that post by clicking on the three dots and pressing report content!

**Comment by Subroutine7901** [comment]
I already did but thanks for the heads up of the other people who were also
pinged
